src/App.css

```diff
@@ -1,66 +0,0 @@
-.App {
-  text-align: center;
-  padding:20px;
-  width:100vh;
-  height: 100vh;
-}
-
-.App-logo {
-  height: 5vmin;
-  background-color: cornflowerblue;
-  font-size:50px;
-  justify-content: center;
-}
-
-.Card-Container {
-  background-color: #366a72;
-  display: flex;
-  flex-wrap: wrap;
-  width: 100vh;
-
-}
-
-.App-link {
-  color: #61dafb;
-}
-h2{
-  font-size: 24px;
-  color:black;
-  margin:20px;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-}
-h3{
-  font-size:16px;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  margin:16px;
-  color:black
-}
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
-.card{
-  background-color: #61dafb;
-  margin:.5rem;
-  padding: 1rem;
-  transition: transform 250ms;
-  border-radius: .5em;
-  justify-content: center;
-
-}
-.card:hover{
```

Comment:
and this?

Reply:
It will slightly grow in size on hover and using the transition time defined in card